reconcile/openshift_resources_base.py

```diff
@@ -1212,7 +1213,7 @@ def _early_exit_fetch_resource(spec: Sequence, settings: Mapping) -> dict[str, s
         c = resource["resource"].get("content")
     del resource["resource"]
     resource[IDENTIFIER_FIELD_NAME] = id
-    content_sha = hashlib.md5(c.encode("utf-8")).hexdigest()
+    content_sha = hashlib.md5(str(c).encode("utf-8")).hexdigest()
```

Comment:
md5 is used here as well? I need to note this down and sync with validator changes if we want to get rid of deprecate warnings of md5. https://docs.astral.sh/ruff/rules/hashlib-insecure-hash-function/